docs/retry-activities-and-workflows.md

```diff
@@ -0,0 +1,185 @@
+---
+id: retry-activities-and-workflows
+title: How to retry Activities and Workflows
+sidebar_label: Retry Activities and Workflows
+---
+
+Temporal is built to expect failures from Activities and Workflows. Retrying Activity or Workflow executions is part of what makes the Temporal system fault tolerant.
```

Comment:
I think the first sentence is definitely misleading but I also incorrect (not 100% sure). 

'Retrying Activity or Workflow executions is part of what makes the Temporal system fault tolerant.' - not really sure this is true

I would say timeouts and retries are a reliable way of handling potential failures. Workflows themselves cannot really fail.